### PR TITLE
Log load time metrics

### PR DESCRIPTION
**Pull request by vdantu**
*Issue #, if available:*
There is a requirement to push the load-time metrics for each model. 

## Output
After changes, the mms_metrics.log contains the following
```
HostName=hostNAme
StartTime=1542747921
Program=MXNetModelServer
Metrics=W-9000-squeezenet=2604 ms Level|Worker
EOE
HostName=hostName
StartTime=1542747921
Program=MXNetModelServer
Metrics=W-9001-squeezenet=2663 ms Level|Worker
EOE
HostName=hostName
StartTime=1542747921
Program=MXNetModelServer
Metrics=W-9003-squeezenet=2689 ms Level|Worker
EOE
```
*Description of changes:*

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java [hunk 1]
```diff
@@ -97,6 +103,17 @@ public WorkerThread(
...
         lifeCycle = new WorkerLifeCycle(configManager, model);
         replies = new ArrayBlockingQueue<>(1);
+        workerLoadTime =
+                new Metric(
+                        "W-"
+                                + this.port
+                                + '-'
+                                + model.getModelName()
+                                        .substring(0, Math.min(model.getModelName().length(), 25)),
+                        Long.toString(System.currentTimeMillis()),
```
Comment (frankfliu) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]: Stri
```suggestion
                        String.valueOf(System.currentTimeMillis()),
```

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java [hunk 2]
```diff
@@ -288,9 +305,16 @@ public void shutdown() {
 
     void setState(WorkerState newState) {
         listener.notifyChangeState(model.getModelName(), newState);
+        long timeTaken = System.currentTimeMillis() - this.startTime;
```
Comment (frankfliu) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]:
```suggestion
        long timeTaken = System.currentTimeMillis() - startTime;
```

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java [hunk 3]
```diff
@@ -288,9 +305,16 @@ public void shutdown() {
...
         if (state != WorkerState.WORKER_TERMINATED) {
             this.state = newState;
         }
+
+        if (state == WorkerState.WORKER_MODEL_LOADED) {
+            workerLoadTime.setValue(Long.toString(timeTaken));
```
Comment (frankfliu) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]:
```suggestion
            workerLoadTime.setValue(String.valueOf(timeTaken));
```

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java [hunk 4]
```diff
@@ -288,9 +305,16 @@ public void shutdown() {
...
+            workerLoadTime.setTimestamp(Long.toString(System.currentTimeMillis() / 1000));
```
Comment (frankfliu) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]:
```suggestion
            workerLoadTime.setTimestamp(String.valueOf(System.currentTimeMillis() / 1000));
```

Comment (piyushghai) [inline frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]: If the objective of the ```time/1000``` operation here is to convert to seconds, then you can use something like this : long seconds = TimeUnit.MILLISECONDS.toSeconds(millis); 

It does the same thing, but makes it easier to read the code.

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java (hunk 1 above)
Comment (vdantu) [reply frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]: I thought it was better to use the objects-type's toString method. Why is this better?

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java (hunk 4 above)
Comment (vdantu) [reply frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]: Sure. I agree. I can make this change.

File: frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java (hunk 1 above)
Comment (frankfliu) [reply frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java]: String.valueOf() has all the overload for most of common types.
Use String.valueOf() will be uniformed for all data types.

**Pull request merged**